src/main/java/frc/robot/subsystems/DriveBase2020.java

```diff
@@ -58,7 +58,7 @@ public double getMotorCurrent() {
     public boolean isMotorLimitActive() {
         //Check if motor current limiting is active (is current draw over or at current limit)
         if (motorCurrent >= Config.CONTIN_CURRENT_AMPS) {
-            motorLimitActive = true;
+            motorLimitActive = true; //For driver feedback purposes only.

```

Comment:
Does this mean the drivetrain current limiting is on by default?

Reply:
No, motorLimitActive is meant for driver feedback, not current limiting.

motorLimitActive is false by default. motorLimitActive will be set to true if motorCurrent rises above CONTIN_CURRENT_AMPS (the continuous current limit).

I'll rewrite my comments to improve clarity.